tests/helpers/providerStateHelper.js

```diff
@@ -286,6 +286,19 @@ const givenTagIsAssignedToFile = (provider, username, password, fileName, tagNam
     .given('a tag is assigned to a file', { username, password, fileName, tagName })
 }
 
+/**
+ * uploads file using TUS protocol
+ *
+ * @param {object} provider
+ * @param {string} username
+ * @param {sring} password
+ * @param {sring} fileName
+ */
+const givenFileUploadedWithTUS = (provider, username, password, fileName) => {
```

Comment:
why do we need that a given step? if its a given action then we don't care how the file is uploaded, if we want to test the TUS upload then it should not be a `given` step

Reply:
@individual-it the given test is not for TUS upload, it is to list directory and check TUS support information.